source/_integrations/sensor.markdown

```diff
@@ -20,6 +20,15 @@ Sensors are a basic integration in Home Assistant. They monitor the states and c
 
 Some of these sensors are built-in to Home Assistant, some are created automatically when you add an integration (see this [list](/integrations/#sensor)), and some can be created manually. The [Statistics](/integrations/statistics) and [Template](/integrations/template) sensors are two examples of the last case.
 
+## The state of a sensor entity
+
+The state of a sensor entity is its currently detected value, which can be either text or a number. If the sensor device is currently unavailable, it will have 'unavailable' as state and when the current value is not yet known, the state will be 'unknown'.
```

Comment:
The use of single quotes is odd here

```suggestion
The state of a sensor entity is its currently detected value, which can be either text or a number. If the sensor device is currently unavailable, it will have `unavailable` as state and when the current value is not yet known, the state will be `unknown`.
```

This addition from @marcelveldt feels weird. Why would we add this here and not to all other entities?

Reply:
https://github.com/home-assistant/home-assistant.io/pull/34809